.github/workflows/deploy-elements-canary.yml

```diff
@@ -0,0 +1,45 @@
+name: Deploy @carbon/ibmdotcom-elements storybook to IBM Cloud (Canary)
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
```

Comment:
Not as familiar with GHA, but wondering if we could separate out some of these steps to be more generic in case we decide to move the builds of other test environments in here, so it doesn't get too repetative? Also, in that regard, the name `build` might be more generic and maybe name this job something specific to the elements test environment build, so later on we can roll in react, etc. Just a thought though!